scripts/post-changelog.mjs

```diff
@@ -1,17 +1,15 @@
 import { $, fetch, argv } from "zx";
 import dotenv from "dotenv-safe";
-import conventionalChangelog from "conventional-changelog";
```

Comment:
This dep is still used in the _old_ publish script, otherwise, it could be removed. It's a dev dep, so not a big deal.

Reply:
Feel free to remove as a part of this PR